discover-cpu-partitioning.py

```diff
@@ -122,26 +129,17 @@ def main():
         if len(housekeeping_cpus) == 0:
             print("this is odd, there are no housekeeping cpus")
         else:
-            print("housekeeping cpus: %s" % (housekeeping_cpus))
-            print("housekeeping cpus: %s" % (system_cpu_topology.formatted_cpu_list(housekeeping_cpus)))
-            print("housekeeping cpus: %d" % (len(housekeeping_cpus)))
-            print("")
-            print("isolated cpus:     %s" % (rcu_nocbs_cpus))
-            print("isolated cpus:     %s" % (system_cpu_topology.formatted_cpu_list(rcu_nocbs_cpus)))
-            print("isolated cpus:     %d" % (len(rcu_nocbs_cpus)))
+            output_cpu_info("housekeeping", housekeeping_cpus)
+
+            output_cpu_info("isolated", rcu_nocbs_cpus)
     elif t_global.args.environment == "container":
         system_cpus = system_cpu_topology(debug = t_global.args.debug)
 
         all_cpus = system_cpus.get_all_cpus()
-        print("all cpus: %s" % (all_cpus))
-        print("all cpus: %s" % (system_cpu_topology.formatted_cpu_list(all_cpus)))
-        print("all cpus: %d" % (len(all_cpus)))
-        print("")
+        output_cpu_info("all", all_cpus)
+
         online_cpus = system_cpus.get_online_cpus()
-        print("online cpus: %s" % (online_cpus))
-        print("online cpus: %s" % (system_cpu_topology.formatted_cpu_list(online_cpus)))
-        print("online cpus: %d" % (len(online_cpus)))
-        print("")
+        output_cpu_info("online", online_cpus)
```

Comment:
this block is common to process and container envs. You might want to save a few lines of code.

Reply:
Yeah, I was waiting until I'm absolutely sure it is going to stay this way before consolidating...